demos/device_defender_for_aws/metrics_collector/lwip/metrics_collector.c

```diff
@@ -0,0 +1,209 @@
+/* Standard includes. */
+#include <stdint.h>
+/* Lwip includes. */
+#include "lwip/arch.h"
+#include "lwip/stats.h"
+#include "lwip/tcpip.h"         /* #define LOCK_TCPIP_CORE()     sys_mutex_lock(&lock_tcpip_core) */
+#include "lwip/ip_addr.h"       /*ip_addr_t, ipaddr_ntoa, ip_addr_copy */
+#include "lwip/tcp.h"           /* struct tcp_pcb */
+#include "lwip/priv/tcp_priv.h" /*tcp_listen_pcbs_t */
```

Comment:
cool thanks for showing what variables these map to.
```suggestion
#include "lwip/ip_addr.h"       /* ip_addr_t, ipaddr_ntoa, ip_addr_copy */
#include "lwip/tcp.h"           /* struct tcp_pcb */
#include "lwip/priv/tcp_priv.h" /* tcp_listen_pcbs_t */
```